README.md

```diff
@@ -14,7 +14,7 @@ On macOS, you can install with [Homebrew](https://brew.sh):
 
 ```bash
 brew tap buildkite/cli
-brew install bk
+brew install buildkite/cli/bk
```

Comment:
I don't think we need the `brew tap buildkite/cli` if we're specifying `buildkite/cli/bk` for the install.
I'm trying to test it, but have a 109 hour ETA Xcode update blocking me from using homebrew…

Reply:
Yep — `brew install buildkite/cli/bk` implicitly taps `buildkite/cli`:

```
pda@paulbookpro ~ $ brew install buildkite/cli/bk
==> Tapping buildkite/cli
Cloning into '/usr/local/Homebrew/Library/Taps/buildkite/homebrew-cli'...
remote: Enumerating objects: 60, done.
remote: Total 60 (delta 0), reused 0 (delta 0), pack-reused 60
Receiving objects: 100% (60/60), 9.30 KiB | 3.10 MiB/s, done.
Resolving deltas: 100% (35/35), done.
Tapped 1 formula (14 files, 16.9KB).
==> Downloading https://github.com/buildkite/cli/releases/download/v1.1.0/bk-darwin-amd64-1.1.0
Already downloaded: /Users/pda/Library/Caches/Homebrew/downloads/eecc454ab2e600238e653f87f8c9dfd78a9c51a504c1d04d626074fe4681aa99--bk-darwin-amd64-1.1.0
==> Installing bk from buildkite/cli
…
```

Reply:
@pda Cool, thanks - I've removed the unecessary command. Just gotta wait for the build and will merge :)